src/util-file.c

```diff
@@ -493,13 +529,25 @@ int FileAppendData(FileContainer *ffc, const uint8_t *data, uint32_t data_len)
 
     if (FileStoreNoStoreCheck(ffc->tail) == 1) {
 #ifdef HAVE_NSS
-        /* no storage but forced md5 */
+        /* no storage but forced md5, sha1 or/and sha256 */
         if (ffc->tail->md5_ctx) {
             if (ffc->tail->md5_ctx)
```

Comment:
I have replicated this bellow, but I'm unsure why it has two if statements with the same condition. Shouldn't this be only one if?